README.md

```diff
@@ -1 +1,70 @@
-# tdr-grafana
\ No newline at end of file
+# TDR Grafana
+
+Repository containing the configuration necessary to setup a Grafana instance for the TDR project.
+
+There is a single TDR Grafana instance running in the TDR management AWS Account.
+
+The ECS cluster is created by Terraform, and runs on the Jenkins vpc on the TDR management AWS account.
+
+For further documentation on the TDR Jenkins configuration see here: https://github.com/nationalarchives/tdr-jenkins
+
+## Project components
+
+### Terraform
+
+This creates the following AWS resources:
+* Subnets
+* ECS cluster
+* ECS service
+* ECS task definition
+* Security groups
+* AWS SSM parameters
+* Database (postgres)
+* IAM permissions for the ECS task role to read the necessary metrics from the different TDR environments (management, integration, production and staging)
+
+The Terraform uses some shared components from the tdr-terraform-modules repository: https://github.com/nationalarchives/tdr-terraform-modules
+
+The Grafana instance runs on the existing Jenkins Vpc in the TDR management AWS account.
+
```

Comment:
Tiny thing but you've got Vpc here and vpc further up. I've had a quick look and AWS uses all capitals so could we stick with that for both to be consistent?

Reply:
updated